_rules/text-contrast-afw4f7.md

```diff
@@ -169,6 +174,26 @@ This dark gray text has a contrast ratio of 12.6:1 on the white background in a
 </script>
 ```
 
+#### Passed Example 10
+
+This text is part of a widget because it is a child of a `button` element. The text has the default
+browser button text color on the default browser button background color. By default, this is black text on a
+light gray background with a contrast ratio of 18.26:1
```

Comment:
I don't think we need to mention that the text is part of a widget since the rule doesn't care.
```suggestion
This text is using the default user agent colors for button, a black (`#000`) text with light gray (<insert value>) background resulting in a contrast ratio of 18.26:1
```
(I also couldn't find the actual colors in [Rendering](https://html.spec.whatwg.org/multipage/rendering.html) in the HTML living standard; this is a non-normative part.)